src/main/java/soconnect/storage/TodoListStorage.java

```diff
@@ -0,0 +1,45 @@
+package soconnect.storage;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Optional;
+
+import soconnect.commons.exceptions.DataConversionException;
+import soconnect.model.ReadOnlyTodoList;
+
+/**
+ * Represents a storage for {@link soconnect.model.TodoList}.
+ */
+public interface TodoListStorage {
+
+    /**
+     * Returns the file path of the data file.
+     */
+    Path getTodoListFilePath();
+
+    /**
+     * Returns TodoList data as a {@link ReadOnlyTodoList}.
+     *   Returns {@code Optional.empty()} if storage file is not found.
+     * @throws DataConversionException if the data in storage is not in the expected format.
+     * @throws IOException if there was any problem when reading from the storage.
+     */
```

Comment:
```suggestion
     * Returns TodoList data as a {@link ReadOnlyTodoList}.
     * Returns {@code Optional.empty()} if storage file is not found.
     *
     * @throws DataConversionException If the data in storage is not in the expected format.
     * @throws IOException If there was any problem when reading from the storage.
     */
```

A new line between the description and the tags.